usl_pipeline/usl_lib/usl_lib/storage/file_names.py

```diff
@@ -10,3 +10,5 @@
 CITYCAT_BUILDINGS_TXT = "Buildings.txt"
 CITYCAT_GREEN_AREAS_TXT = "GreenAreas.txt"
 CITYCAT_SPATIAL_GREEN_AREAS_TXT = "Spatial_GreenAreas.txt"
+
+WPS_DOMAIN3_NC_REGEX = "met_em.d03.*.nc$"
```

Comment:
I think you'd need to escape those `.`s which are meant to be periods? You can also use r-strings to avoid Python escaping on to of regex escaping.
```python
WPS_DOMAIN3_NC_REGEX = r"met_em\.d03.*\.nc$"
```